extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -241,21 +244,26 @@ export class CreateSqlMigrationServiceDialog {
 			required: true,
 			editable: true,
 			fireOnTextChange: true,
+			CSSStyles: {
+				'margin-top': '-1em'
+			}
```

Comment:
can this be done with out using a -1em margin value?  it is usually an indicator that the above / outside item needs to change the padding or margin.

Reply:
The actual text of a text component has a default `margin: 1em 0`. Text components also can have required * indicators and tooltips. 

There currently isn't a way to modify the margin of the text directly.

To remove the gap between the input label and the inputBox/dropdown, I added the `margin-top: -1em` for those input elements. Do you know of another way we can do this?

![image](https://user-images.githubusercontent.com/14362577/132064096-5a899f6a-80db-4f9a-8a5b-13cb1533bed9.png)

Reply:
I think we can invest in cleaning up the label/input components by creating a helper function to keep the styling consistent.